XenAdmin/Wizards/ImportWizard/ImportWizard.cs

```diff
@@ -310,7 +310,7 @@ protected override void UpdateWizardContent(XenTabPage page)
                 _targetConnection = m_pageHost.SelectedTargetPool?.Connection;
                 var oldHostSelection = m_vmMappings.ToDictionary(kvp => kvp.Key, kvp => kvp.Value.TargetName);
                 m_vmMappings = m_pageHost.VmMappings;
-
+                m_pageFinish.CanStartVmsAutomatically = m_pageHost.ApplianceCanBeStarted;

```

Comment:
The host selection on the host page serves only for showing local SRs on the SR selection page. This means that the user may have selected a host, but on the SR page they may have eventually opted for shared SR, which means that the VM can still start on a different host with adequate memory or CPUs and we shouldn't disablel the check box (as long as it is a valid case to have a pool with mixed CPU numbers, if I remember well XC does not inlcude the number in the CPU checks when joining pools, we need to double check xapi allows it too). 

P.S. When I fixed the wording on the host page, I should have removed the Target from the summary page because it's not useful and is also causing confusion. I can send a PR if you want.